app/services/remove_school_from_virtual_cap_pool_service.rb

```diff
@@ -0,0 +1,56 @@
+class RemoveSchoolFromVirtualCapPoolService
+  attr_reader :school, :rb
+
+  delegate :laptop_allocation_id, :router_allocation_id, to: :school
+
+  def initialize(school, responsible_body)
+    @school = school
+    @rb = responsible_body
+  end
+
+  def call
+    remove_school! if school.in_virtual_cap_pool?
+  rescue StandardError => e
+    failed(e)
+  end
+
+private
+
+  def remove_device_allocation_from_pool!(allocation_id)
+    SchoolVirtualCap.find_by(school_device_allocation_id: allocation_id)&.destroy!
+    # &.recalculate_caps!
```

Comment:
Worth deleting that comment?

Reply:
Yes, sure. I didn´t yet to do a final check that I forgot to do. I'll do it now and remove the comment after double checking we don't need it.